external/js/cothority/src/network/connection.ts

```diff
@@ -153,7 +164,7 @@ export class WebSocketConnection implements IConnection {
 
         return new Observable((sub) => {
             const url = new URL(this.url.href);
-            url.pathname = `/${this.service}/${message.$type.name.replace(/.*\./, "")}`;
+            url.pathname =  `${url.pathname}${this.service}/${message.$type.name.replace(/.*\./, "")}`;
```

Comment:
Now we can use it as it always ends with a slash :)

```suggestion
            url.pathname += `${this.service}/${message.$type.name.replace(/.*\./, "")}`;
```